model/src/main/java/org/jboss/pnc/model/BuildRecord.java

```diff
@@ -164,6 +164,14 @@
     @Setter
     private String sshPassword;
 
+    @Getter
+    @Setter
+    private String executionRootName;
```

Comment:
What should be the executionRootName and where do we get it from?
Can you add some comments. (the same for version)